### PR TITLE
Bump dependencies

### PR DESCRIPTION
**Pull request by guersam**
I'd also suggest enabling [scala-steward](https://github.com/fthomas/scala-steward) that helps keep dependencies up-to-date.

### CODE REVIEW

**Comment by guersam** [comment]
Something's wrong with Travis :thinking: 
https://travis-ci.community/t/error-installing-oraclejdk8-expected-feature-release-number-in-range-of-9-to-14-but-got-8/3766

File: modules/tests/src/test/scala/aecor/tests/ScheduleEventCodecSpec.scala [hunk 1]
```diff
@@ -13,7 +14,8 @@ class ScheduleEventCodecSpec extends AkkaSpec with PropertyChecks {
   val decoder = PersistentDecoder[ScheduleEvent]
 
   implicit val arbitraryLocalDateTime = Arbitrary(Gen.lzy(Gen.const(LocalDateTime.now())))
-  implicit val arbitraryInstant = Arbitrary(Gen.lzy(Gen.const(Instant.now())))
+  implicit val arbitraryInstant =
+    Arbitrary(Gen.lzy(Gen.const(Instant.now().`with`(ChronoField.MICRO_OF_SECOND, 0L))))
```
Comment (guersam) [inline modules/tests/src/test/scala/aecor/tests/ScheduleEventCodecSpec.scala]: https://travis-ci.org/notxcain/aecor/jobs/548042560#L554-L560

OpenJDK 9+ offers a more precise system clock, but `PersistentEncoder[ScheduleEvent]` uses the milliseond only. For backward compability, I just dropped microseconds in the spec. WDYT?
https://bugs.openjdk.java.net/browse/JDK-8068730

File: modules/tests/src/test/scala/aecor/tests/AkkaPersistenceRuntimeSpec.scala [hunk 2]
```diff
@@ -45,7 +45,7 @@ class AkkaPersistenceRuntimeSpec
 
   override def systemName = system.name
 
-  override implicit val patienceConfig = PatienceConfig(15.seconds, 150.millis)
+  override implicit val patienceConfig = PatienceConfig(30.seconds, 150.millis)
```
Comment (guersam) [inline modules/tests/src/test/scala/aecor/tests/AkkaPersistenceRuntimeSpec.scala]: It takes much longer now... Maybe due to OpenJDK?

**Comment by notxcain** [comment]
Hi! Thanks for the PR. I'm not sure that Akka Persistence Cassandra `0.9х] is compatible with ]0.6х]

**Comment by guersam** [comment]
You're right. Migration is required to upgrade... I'll revert it for now. https://github.com/akka/akka-persistence-cassandra/blob/master/docs/src/main/paradox/migrations.md#migrations-to-080-and-later

File: modules/tests/src/test/scala/aecor/tests/ScheduleEventCodecSpec.scala (hunk 1 above)
Comment (notxcain) [reply modules/tests/src/test/scala/aecor/tests/ScheduleEventCodecSpec.scala]: I'm fine with that. We did the same in an internal branch.

Comment (notxcain) [reply modules/tests/src/test/scala/aecor/tests/ScheduleEventCodecSpec.scala]: We used `.truncatedTo(ChronoUnit.MILLIS)` WDYT?

**Comment by notxcain** [comment]
@guersam I resolved some conflicts with `master`

**Comment by notxcain** [comment]
Thank you @guersam! Merging now 🚀

**Pull request merged**